### DISCUSSION TITLE
How do I build PRG locally?

### DISCUSSION THREAD

**Discussion by TechStackz** [scottgriv/PRG-Personal-Repository-Guidelines#15] [Q&A] [answered]
hi again. sorry..one more question please. 
how can I build PRG locally. is this possible?

**Comment by scottgriv** [comment, marked as answer]
Hi. No worries at all! Feel free to ask anything you'd like.

The answer is yes, you can! Follow the [GitHub Pages Deployment](https://github.com/scottgriv/PRG-Personal-Repository-Guidelines/blob/main/categories/table_generator_guide.md#github-pages-deployment) Local Testing section in the [Table Generator Guide](https://github.com/scottgriv/PRG-Personal-Repository-Guidelines/blob/main/categories/table_generator_guide.md) for information on how to do this.

FYI: You will need Jekyll and Python installed locally to do this. I'll add that to the guide if I didn't clarify that. Thanks again!
